Framework/DataHandling/src/LoadEMU.cpp

```diff
@@ -717,11 +875,135 @@ void LoadEMU::loadDopplerParameters(API::LogManager &logm) {
         180.0 - asin(0.001 * doppThreshold / m_dopplerAmpl) * 180.0 / M_PI +
         doppDelay * m_dopplerFreq;
   }
-  logm.addProperty<double>("DopplerPhase", m_dopplerPhase);
+
+  // problem adding 'bool' to log
+  int32_t calPhase = (m_calibrateDoppler ? 1 : 0);
+  logm.addProperty("CalibratePhase", calPhase);
+}
+
+/// Calibrate the doppler phase based on the analysed events using
+/// the \p eventCounts and \p eventVectors.
+template <typename FD>
+void LoadEMU<FD>::calibrateDopplerPhase(
+    std::vector<size_t> &eventCounts,
```

Comment:
`const` vectors?

Reply:
Yes, changed - apologies for all these missing consts.